.github/workflows/build-and-release.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+          architecture: x64
```

Comment:
is it necessary, since we are specifying the python versions?

Reply:
Arch is an action setting - for now, it is the default architecture. But if they change the default architecture our action might stop working, it is why I left it. And actually, for each python version, there is a couple of arch available https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json